string name
bool cvWindow
---
string personId
string features
